pkgs/development/python-modules/docker_compose/default.nix

```diff
@@ -41,7 +41,7 @@ buildPythonApplication rec {
     homepage = https://docs.docker.com/compose/;
     description = "Multi-container orchestration for Docker";
     license = licenses.asl20;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
the whole line should be removed instead as it is already set to this value by `buildPythonPackage`

Reply:
done